tests/test_workspace.py

```diff
@@ -887,3 +887,18 @@ def test_workspace_poiless(datadir):
 
     assert model.config.poi_name is None
     assert model.config.poi_index is None
+
+
+def test_workspace_without_validation(mocker):
+    # TODO: refactor if #1709 is accepted / merged
```

Comment:
#1709 refactors the `simplemodels` setup to use a fixture instead, so this could be adopted here too for consistency.

Reply:
Yup, this all sounds good. We can merge PR #1709 first and then rebase this one on top.

Reply:
Done now in e737e4d.